lib/ansible/modules/cloud/amazon/rds.py

```diff
@@ -1124,6 +1124,9 @@ def main():
         'restore': restore_db_instance,
     }
 
+    if module.params.['command'] != 'modify' and module.params.['multi_zone'] is None:
+        module.params.['multi_zone'] = False
```

Comment:
Having said that, I don't believe these two lines are required at all - we should only use multi_zone if it's explicitly set, and let the boto rds2 API handle the default (which it does - if you pass it multi_az=None (it's default), you'll get a non multi AZ RDS instance)